Introduce `$permissions` modifier description (#40)
AdguardTeam/KnowledgeBase: docs/general/ad-filtering/create-own-filters.md

>
> Rules with `$csp` modifier are not supported by AdGuard Content Blocker, AdGuard for iOS and Safari.

#### **`$permissions`** {#permissions-modifier}

This modifier completely changes the rule behavior. If it is applied to a rule, it will not block the matching request. The response headers are going to be modified instead.

> In order to use this type of rules, it is required to have the basic understanding of the [Feature Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Feature_Policy) security layer.

For the requests matching a `$permissions` rule, AdGuard strengthens response's feature policy by adding additional feature policy equal to the `$permissions` modifier contents. `$permissions` rules are applied independently from any other rule type. Other basic rules have no influence on it **save for document-level exceptions** (see the examples section).

> **Multiple rules matching a single request.**
>
> In case if multiple `$permissions` rules match a single request, AdGuard will apply each of them.

**Syntax**

> `$permissions` value syntax is similar to the `Permissions-Policy` (or `Feature-Policy`) header syntax.

Reviewer: We need a list of permissions here (at least the most prominent ones + a link to the full list)

Author: Added some more links. Not sure about the most prominent of the directives as most of them are still experimental. And besides, I suppose the list is still subject to change.

> In order to use this type of rules, it is required to have the basic understanding of the [Feature Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Feature_Policy) security layer.

For the requests matching a `$permissions` rule, AdGuard strengthens response's feature policy by adding additional feature policy equal to the `$permissions` modifier contents. `$permissions` rules are applied independently from any other rule type. Other basic rules have no influence on it **save for document-level exceptions** (see the examples section).

> **Multiple rules matching a single request.**
>
> In case if multiple `$permissions` rules match a single request, AdGuard will apply each of them.

**Syntax**

`$permissions` value syntax is similar to the `Permissions-Policy` (or `Feature-Policy`) header [syntax](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy).
See [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy#directives) for the list of the available directives.

> `$permissions` value can be empty in the case of exception rules. See examples section for further information.

Reviewer 2: ```suggestion
`$permissions` value can be empty in the case of exception rules — see examples below.
```

> **Limitations**
>

> > 1. Please note that there are a few characters forbidden in the `$permissions` value: `,`, `$`;

Reviewer 2: ```suggestion
> 1. Two characters are forbidden in the `$permissions` value: `,` and `$`;
```

> > 2. `$permissions` is compatible with the limited list of modifiers: `$domain`, `$important`, `$subdocument`.

Reviewer 2: ```suggestion
> 2. `$permissions` is compatible with the limited list of modifiers: `$domain`, `$important`, and `$subdocument`.
```

**Syntax**

`$permissions` value syntax is similar to the `Permissions-Policy` (or `Feature-Policy`) header [syntax](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy).

> See [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy#directives) for the list of the available directives.

Reviewer 2: ```suggestion
The list of the available directives is available [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy#directives).
```
